react/features/lobby/components/native/LobbyModeButton.js

```diff
@@ -63,14 +64,16 @@ class LobbyModeButton extends AbstractButton<Props, any> {
  * @param {Props} ownProps - The own props of the component.
  * @returns {Props}
  */
-export function _mapStateToProps(state: Object): $Shape<Props> {
+export function _mapStateToProps(state: Object, ownProps: Props): $Shape<Props> {
     const conference = getCurrentConference(state);
     const { lobbyEnabled } = state['features/lobby'];
     const lobbySupported = conference && conference.isLobbySupported();
+    const flag = getFeatureFlag(state, LOBBY_MODE_ENABLED, true);
+    const { visible = flag } = ownProps;
```

Comment:
Doesn't is mean that the button will be visible if an explicit visible prop is passed to the button even if the feature should be disabled?

Also, I think feature toggling means a more than hiding a button. This way we just replicate the interfaceConfig settings. Can you please consider adding some more checks?